src/redux/actions/dictionaries/dictionaryActionCreators.js

```diff
@@ -120,8 +120,9 @@ export const removeDictionaryConcept = (data, type, owner, collectionId) => disp
       );
     }
     )
-    .catch(() => {
-      notify.show("Network Error. Please try again later!", 'error', 6000)
+    .catch((error) => {
+      error.response ? notify.show(error.response.data.detail, 'error', 6000):
```

Comment:
What is difference between what these two lines are doing? 124 and 125

Reply:
Line 24 checks if there is a response from the server and it displays that response, for example, it could be permission denied response. If the check on line 24 is false, we display the network error message(line 25) because it confirms an issue on the client side. Previously, it was showing a network error message despite it being a server response.